internal/server/audit/logfile/logfile_test.go

```diff
@@ -17,7 +17,7 @@ func TestNewSink_NewFile(t *testing.T) {
 	var (
 		logger = zap.NewNop()
 		path   = os.TempDir()
-		file   = path + "audit.log"
+		file   = path + "/audit.log"
```

Comment:
Maybe we can just do this going forward and let Go figure it out:
```suggestion
		file   = filepath.Join(path, "audit.log")
```